src/MediaPlayer.elm

```diff
@@ -349,67 +365,44 @@ encodeVideoEvent event =
     case event of
         Setup ->
             Encode.object
-                [ "kind" => Encode.string "setup" ]
+                [( "kind", Encode.string "setup" )]
 
         Play ->
             Encode.object
-                [ "kind" => Encode.string "play" ]
+                [( "kind", Encode.string "play" )]
 
         Pause ->
             Encode.object
-                [ "kind" => Encode.string "pause" ]
+                [( "kind", Encode.string "pause" )]
 
         Stop ->
             Encode.object
-                [ "kind" => Encode.string "stop" ]
+                [ ("kind", Encode.string "stop" )]
 
         Restart ->
             Encode.object
-                [ "kind" => Encode.string "restart" ]
+                [( "kind", Encode.string "restart" )]
 
         Mute ->
             Encode.object
-                [ "kind" => Encode.string "mute" ]
+                [( "kind", Encode.string "mute" )]
 
         Unmute ->
             Encode.object
-                [ "kind" => Encode.string "unmute" ]
+                [ ("kind", Encode.string "unmute" )]
 
         VolumeDown ->
             Encode.object
-                [ "kind" => Encode.string "volumedown" ]
+                [( "kind", Encode.string "volumedown" )]
 
         VolumeUp ->
             Encode.object
-                [ "kind" => Encode.string "volumeup" ]
+                [ ("kind", Encode.string "volumeup" )]
 
         SeekTo position ->
             Encode.object
-                [ "kind" => Encode.string "seekto"
-                , "position" => Encode.float position
+                [( "kind"
+                , Encode.string "seekto")
+                , ("position"
+                , Encode.float position)
```

Comment:
This looks a bit odd, like there are four elements in the list rather than two tuples. Please use elm-format to format the file, then resubmit.

Reply:
Ups, dunno why it didnt do that on save.. did it manually now :+1: